app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<%= stylesheet_link_tag "application"%>
+
+<div class="logger_in_form"><p><%= notice %></p>
+
+<h1>Recipes</h1>
+
+<div id="recipes">
+  <% @recipes.each do |recipe| %>
+    <%= render recipe %>
+    <p><%= link_to 'Show the recipe', recipe %></p>
+  <% end %>
+</div>
+
+<%= link_to 'New Recipe', new_recipe_path %>
```

Comment:
- [ ] Kindly make this view matches the given wireframe.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/209049153-72c2a7ae-ef18-4ce0-adf4-3138178944d8.png) | ![image](https://user-images.githubusercontent.com/68796232/209049120-a743ae62-e0a2-4f65-ba77-7b6c91ca8fff.png) |

Reply:
thank you for this